pkg/kubectl/cmd/get.go

```diff
@@ -220,13 +222,23 @@ func RunGet(f *cmdutil.Factory, out io.Writer, cmd *cobra.Command, args []string
 	}
 
 	// use the default printer for each object
+	printer = nil
+	var lastMapping *meta.RESTMapping
+	w := tabwriter.NewWriter(out, kubectl.TabwriterMinWidth, kubectl.TabwriterWidth, kubectl.TabwriterPadding, kubectl.TabwriterPadChar, kubectl.TabwriterFlags)
```

Comment:
hmm... reusing the writer that way is gorpy in two places... any cleaner way to accomplish that?